lib/ansible/modules/packaging/language/pip.py

```diff
@@ -235,6 +237,61 @@
                              'pip': 'import pkg_resources; print(pkg_resources.get_distribution("pip").version)'}
 
 
+class Package(Requirement):
+    def __init__(self, name_string, version_string=None):
+        if version_string is not None:
+            if version_string[0].isdigit():
+                name_string = "%s ==%s" % (name_string, version_string)
+            else:
+                name_string = "%s %s" % (name_string, version_string)
+        super(Package, self).__init__(name_string)
```

Comment:
Apparently, it will not work well under Python 2.6.9. It seems to be requiring you to call `Requirement.parse()` to construct an instance of this class. So we'll need to shuffle a couple of things around to make it work.